esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
+                method: "post"
+            }).then(function(data){
+                if (data){
+                    document.cookie = "ESPSessionID" + location.port + " = '' "; "expires=Thu, 01 Jan 1970 00:00:00 GMT"; // or -1
+                    window.location.reload();
```

Comment:
@GordonSmith the reload (making a call to ESP) will force the user to Login.html until validated. We are trying to simulate some offline/ESP latency issues to send someone to the login page even if there are latency issues.